docs/minio_homebrew.md

```diff
@@ -0,0 +1,35 @@
+# Minio Installation on macOS
+
+## Fresh Install
+
+Install Minio packages for macOS via brew.
+
+```
+brew install minio/stable/minio
+minio server ~/Photos
+```
+
+## Upgrade 
+
+Step 1: Uninstall minio if you installed it using `brew install minio`
+
+```
+brew uninstall minio 
+```
+Step 2: Fresh Install using new path
```

Comment:
All the words can be either capitalized or none. Since this not a header, I think small letters will be better.